QUEST_JOBSTEP.tsv

```diff
@@ -152,29 +152,29 @@ QUEST_JOBSTEP_20150317_000151	I need to gather the energy of fire for my researc
 QUEST_JOBSTEP_20150317_000152	Be careful. The pot will break if you don't keep an eye on it. 
 QUEST_JOBSTEP_20150317_000153	Please put this pot on the ground when you go to the Descended Tree Garden.{nl}After that, when you defeat the enemies nearby, energies will be accumulated in the pot.
 QUEST_JOBSTEP_20150317_000154	There is an important book that I'm looking for. {nl}If you get me that book, I will help you learn the magic of Cryomancers. 
-QUEST_JOBSTEP_20150317_000155	Did you say you'll find the book? {nl}Sorry but you're late. 
+QUEST_JOBSTEP_20150317_000155	$Did you say you'll find the book? {nl}Sorry, but you're late. 
 QUEST_JOBSTEP_20150317_000156	I heard rumors that it's in the Miners' Village, but I don't know the exact location of it. {nl}Maybe the Mayor would know...
 QUEST_JOBSTEP_20150317_000157	The Vubbes took all the books when they attacked. {nl}Maybe they will use it for fire in their outpost. 
 QUEST_JOBSTEP_20150317_000158	If there was something like that, we would have taken care of it. {nl}You go and try looking for it. 
-QUEST_JOBSTEP_20150317_000159	You found it. Great job. {nl}You'll have to work really hard to follow my trainings. 
-QUEST_JOBSTEP_20150317_000160	I heard the book was last seen in the Miners' Village. {nl}It's a very important book so maybe the Mayor might know.
+QUEST_JOBSTEP_20150317_000159	$You found it. Great job. {nl}You'll have to work really hard to follow my training. 
+QUEST_JOBSTEP_20150317_000160	$I heard the book was last seen in the Miners' Village. {nl}It's a very important book, so maybe the Mayor might know.
```

Comment:
"so the Mayor might know." sounds unfinished.
"so the Mayor might know something about it."?